bindings/python/malamute/__init__.py

```diff
@@ -99,5 +101,5 @@ def recv(self):
         m = self.c.recv()
         return (
             self.c.command(), self.c.sender(),
-            self.c.subject(), _zmsg_to_list(m)
+            self.c.subject(), _zmsg_to_list(m) if m else None
```

Comment:
If interrupted, `m` is a null pointer and calling anything on it ends up with assert aborting.